readme.md

```diff
@@ -1,44 +1,60 @@
-    <a href="https://github.com/gmrchk/swup/blob/master/LICENSE"><img src="https://img.shields.io/github/license/gmrchk/swup.svg" alt="License"></a>
-    <a href="https://www.npmjs.com/package/swup"><img src="https://img.shields.io/npm/dt/swup.svg" alt="npm downloads"></a>
-    <a href="https://circleci.com/gh/swup/swup"><img src="https://circleci.com/gh/swup/swup.svg?style=svg" alt="CircleCI"></a>
+  <a href="https://www.npmjs.com/package/swup"><img src="https://img.shields.io/npm/v/swup.svg" alt="npm version"></a>
+  <a href="https://bundlephobia.com/package/swup"><img src="https://img.shields.io/bundlephobia/minzip/swup.svg" alt="Bundle size"></a>
+  <a href="https://github.com/gmrchk/swup/blob/master/LICENSE"><img src="https://img.shields.io/github/license/gmrchk/swup.svg" alt="License"></a>
+  <a href="https://www.npmjs.com/package/swup"><img src="https://img.shields.io/npm/dt/swup.svg" alt="npm downloads"></a>
+  <a href="https://circleci.com/gh/swup/swup"><img src="https://circleci.com/gh/swup/swup.svg?style=shield" alt="Build status"></a>
 </p>
 
-[Here](https://medium.com/@gmarcuk/introducing-swup-v2-814e40316dee)'s what's new in v2. Check [changelog](https://swup.js.org/other/changelog) for recent changes, see all [swup repositories](https://github.com/swup/) to discover more, or checkout the [discussions](https://github.com/swup/swup/discussions).
-
-- [Discussions](https://github.com/swup/swup/discussions)
-- [Getting Started](https://swup.js.org/getting-started)
-  - [Example](https://swup.js.org/getting-started/example)
-  - [Demo](https://swup.js.org/getting-started/demo)
-  - [Reloading JavaScript](https://swup.js.org/getting-started/reloading-javascript)
-  - [Video Tutorials](https://swup.js.org/getting-started/videos)
-- [Options](https://swup.js.org/options)
-- [Events](https://swup.js.org/events)
-- Extensions
-    - [Plugins](https://swup.js.org/plugins)
-    - [Themes](https://swup.js.org/themes)
-    - [3rd Party Integrations](https://swup.js.org/third-party-integrations)
-- [JavaScript API](https://swup.js.org/api)
-- [Swup CLI](https://swup.js.org/cli)
-- [CI/CD Integration](https://swup.js.org/ci-cd)
-- Other
-    - [Changelog](https://swup.js.org//other/changelog)
-    - [Contributions](https://swup.js.org/other/contributions)
-    - [Repositories](https://github.com/swup)
-    - [Sites using swup](https://github.com/swup/swup/discussions/333)
-
-If you're having trouble implementing swup, checkout [Common Issues](https://swup.js.org/other/common-issues), [Closed Issues](https://github.com/gmrchk/swup/issues?q=is%3Aissue+is%3Aclosed) or open a [new one](https://github.com/gmrchk/swup/issues/new).
-
-[Become a backer or sponsor on Open Collective](https://opencollective.com/swup) or support swup through [GitHub sponsors](https://github.com/sponsors/gmrchk). 
+<br>
+
+# Swup
+
+Complete, flexible, extensible and easy to use page transition library for your static website.
```

Comment:
Now that I read that part that I wrote myself, the "static" keyword is a bit misleading, since Swup supports dynamic web as well. Your description with server-side rendered is much better!

```suggestion
Complete, flexible, extensible, and easy-to-use page transition library for your server-side rendered website.
```